components/board/genericlinux/data.go

```diff
@@ -259,13 +293,6 @@ func getPwmChipDefs(pinDefs []PinDefinition) (map[string]pwmChipData, error) {
 				}
 
 				pwmChipsInfo[chipName] = pwmChipData{Dir: chipPath, Npwm: npwm}
-				// Now that we've found the chip info, we need to break out of 2 different for
-				// loops, to go on to the next chip name. This is just the first one so far...
-				break
-			}
-			if found {
-				// ...and this is the second one. We've already found the info for the current
-				// chip name, so move on to the next name.
```

Comment:
I was pulling the latest code in this file and found this to be different from what my branch had., I will find a better way to merge the current data.go and my changes.

Reply:
Ohh, that makes more sense! :sweat_smile: I've found that rebasing off the `main` branch is usually pretty reliable. I'm happy to help you sort this out if you want a second set of eyes.